lib/steps.js

```diff
@@ -641,7 +641,7 @@ Then(/the "([^"]+)" link opens in (a new window|the same window)/, async (linkTe
   let [link] = await scope.page.$x(`//a[contains(text(), "${linkText}")]`);
 
   let prop_obj = await link.getProperty('target');
-  let target = await await prop_obj.jsonValue();
+  let target = await prop_obj.jsonValue();
```

Comment:
FYI I looked into it, and program logic wise, an extra `await` doesn't break anything. However it will wrap the value in a promise, which slows the program down a little bit more than necessary.